components/ChatApp.tsx

```diff
@@ -13,7 +13,8 @@ export class ChatApp extends React.Component<
         user: { id: string; username: string };
     },
     {
-        channels: string[];
+        channelState: any[];
+        channels: any[];
```

Comment:
Why do you switch from string to any?

Reply:
Because befor, channels contained only string, but now it contain the channels informations

Reply:
ok thanks for explanation, I'll take a deeper look later